README.md

```diff
@@ -1,3 +1,3 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+In order to run this app, you must add .evn-development and .env-test files in the main directory. These files should include a PGDATABASE and a PGPASSWORD for sql. The databases should be called nc_news and nc_news_test respectively.
```

Comment:
This could do with being a little clearer. The files would be .env.development not .env-development. You could more more explicit too and say that the .env.development file should have PGDATABASE=nc_news for example.